### ISSUE TITLE
MD037/no-space-in-emphasis triggers when using LaTex block

### ISSUE THREAD

**Issue by bathtown** [DavidAnson/vscode-markdownlint#146]
when I write something like this:

```md
$$
x * y = x * (y_1 + y_2 + \cdots + y_n)
$$
```

it says MD037/no-space-in-emphasis: Spaces inside emphasis markers. But it's just `*` in LaTex block, and I think this should be fixed :)

![image](https://user-images.githubusercontent.com/49479124/102000480-eb40b180-3d22-11eb-9d6d-3af5eef39f2c.png)
